crypto/fipsmodule/ec/.gitattributes

```diff
@@ -1 +1,2 @@
-ecp_nistz256_table.inl linguist-generated=true
+p256-x86_64-table.h linguist-generated=true
+p256_table.h linguist-generated=true
```

Comment:
It seems like all this ECC stuff can be deleted now. I think you can revert all crypto/fipsmodule/ec stuff.